hudi-utilities/src/main/java/org/apache/hudi/utilities/schema/SchemaProviderWithPostProcessorDisableSource.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.utilities.schema;
+
+import org.apache.hudi.common.util.Option;
+
+import org.apache.avro.Schema;
+
+public class SchemaProviderWithPostProcessorDisableSource extends SchemaProviderWithPostProcessor {
+  private final SchemaProvider schemaProvider;
+  private final Option<SchemaPostProcessor> schemaPostProcessor;
+  public static class Config {
+    public static final String DISABLE_POST_PROCESSOR_ON_SOURCE_SCHEMA =
```

Comment:
I feel we should make this generic. Introduce this config as a generic config and apply it to any schema provider.

Reply:
I mean, we should not name this class as SchemaProviderWithPostProcessor"Disable"Source.
Consider this as general schema handling where optionaly post processor could be added for source and target schema.